feature-libs/my-account/organization/occ/converters/occ-org-unit-address-list-normalizer.ts

```diff
@@ -1,22 +1,22 @@
 import { Injectable } from '@angular/core';
-import { B2B_ADDRESS_NORMALIZER } from '@spartacus/my-account/organization/core';
 import {
+  Address,
   Converter,
-  Occ,
-  EntitiesModel,
-  B2BAddress,
   ConverterService,
+  EntitiesModel,
+  Occ,
 } from '@spartacus/core';
+import { B2B_ADDRESS_NORMALIZER } from '@spartacus/my-account/organization/core';
```

Comment:
In this case shouldn't we also drop B2B_ADDRESS_NORMALIZER and just provide ADDRESS_NORMALIZER?